COPY/etc/manageiq/postgresql.conf.d/01_miq_overrides.conf

```diff
@@ -26,6 +26,9 @@ tcp_keepalives_interval = 75
 # RESOURCE USAGE (except WAL)
 #------------------------------------------------------------------------------
 
+shared_buffers = 2GB
+work_mem = 4194kB
+maintenance_work_mem = 512MB
```

Comment:
This is per autovacuum worker. There are 3 of those by default. So if we want this value up this high we should set [autovacuum_work_mem](https://www.postgresql.org/docs/9.5/static/runtime-config-resource.html#GUC-AUTOVACUUM-WORK-MEM) to something a bit smaller I think. Maybe 128MB (twice the default)?

Reply:
Good idea just to be safe. However, this is where most of the benefit of this tuning was realized. These tuning options combined with the truncate PR will be a great combination of improvements. I'll update.